tests/data/stats/mock/statistics/tx_bytes_increased_big

```diff
@@ -0,0 +1 @@
+61224753204
```

Comment:
this solution is not bad, but i think it might be better to create these mock files on the fly in a TemporaryDirectory:

https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory

So request a temp folder during mocking, write some files for the test case and then point the logic to read those files. Don't forget to clean up afterwards. This way we dont need to worry about all these temp files for special use cases.

Thing is, we otherwise need to bundle these files also with the tarball, which requires updating the `MANIFEST.in` file. Otherwise, the files are not bundled with the source tarball and the eventual packages fail to build. I prefer to keep this kind of admin minimal, and generating temp files is cleaner (as long as you remove the temp files afterwards).

You can also create a Mock IO object per `open()`, but that might be more work.

Reply:
Thanks that makes sense!